### ISSUE TITLE
App name on KDE Connect

### ISSUE THREAD

**Issue by xmha97** [filips123/PWAsForFirefox#365]
Hi, I use KDE Connect to use my laptop from my phone.
Everything is working fine but PWA's names are shown wrongly!
For example, the YouTube app is shown as EF02F7C08385651

# Windows
![image](https://github.com/filips123/PWAsForFirefox/assets/23485114/db776225-ed86-4aad-8204-a323476b7838)

# Android (KDE Connect)
![Screenshot_2023-07-12-00-31-33-178_org kde kdeconnect_tp](https://github.com/filips123/PWAsForFirefox/assets/23485114/14bc0198-2c52-4e8b-97b7-3d500aac10c1)

**Comment by filips123** [comment]
Does it display the correct name for normal Firefox?

**Comment by xmha97** [comment]
> Does it display the correct name for normal Firefox?

No, I just noticed that the Firefox name is not showing correctly either!

**Comment by filips123** [comment]
This is probably a problem with Firefox or KDE Connect. You should probably submit a bug report to them (if there is none already), because I think I cannot fix this here.

**Comment by xmha97** [comment]
OK, thanks 🙏
